lang/ghc/Portfile

```diff
@@ -202,6 +202,14 @@ if { [variant_isset "prebuilt"] } {
                     sha256  f444012f97a136d9940f77cdff03fda48f9475e2ed0fec966c4d35c4df55f746 \
                     size    23338772
 
+    # build depends upon these x86_64 binaries
+    platform arm {
```

Comment:
No need to restrict this to arm systems; if the archs of the dependencies don't need to match, then that will be the case on all platforms.

Reply:
Thanks, changed.